charts/rancher-webhook/templates/pre-delete-hook-cluster-role.yaml

```diff
@@ -13,4 +13,10 @@ rules:
     resources: [ "mutatingwebhookconfigurations" ]
     verbs: [ "delete" ]
     resourceNames: [ "rancher.cattle.io"]
+  - apiGroups: [""]
+    resources: ["serviceaccounts"]
+    verbs: ["get"]
+  - apiGroups: ["policy"]
+    resources: ["podsecuritypolicies"]
+    verbs:     ["use"]
```

Comment:
Do you have to specify the name of the PSP to use with `resourceName`?

Reply:
It is optional but I just added it.

Reply:
Didn't realize it was optional.